examples/mocking/README.md

```diff
@@ -1,62 +1,25 @@
-# Interoperability with `gomock`
+# Using the testkit
 
-When developing large applications, you find yourself in need of mocking APIs. For Go, there's [`gomock`](https://github.com/golang/mock) framework for code generating testing mocks. In this small example, we'll show how to use `gomock` with Databricks SDK for Go.
+To test your application it is often necessary to mock external dependency.
+The SDK contains a testkit you can use for such a purpose, so you don't have to 
+use the [`gomock`](https://github.com/golang/mock) framework for code generating 
+testing mocks. 
 
-Please read through [`dbfs_test.go`](dbfs_test.go) test example. 
-
-## Declaring which mocks to generate
-
-```go
-//go:generate go run github.com/golang/mock/mockgen@latest -package=mocks -destination=mocks/dbfs.go github.com/databricks/databricks-sdk-go/service/dbfs DbfsService
-```
-
-* `go run github.com/golang/mock/mockgen@latest` downloads and executes the latest version of `mockgen` command
-* `-package=mocks` instructs to generate mocks in the `mocks` package
-* `-destination=mocks/dbfs.go` instructs to create `dbfs.go` file with mock stubs.
-* `github.com/databricks/databricks-sdk-go/service/dbfs` tells which Databricks package to look services in.
-* `DbfsService` tells which services to generate mocks for.
-
-## Initializing `gomock`
-
-Every test needs the following preamble:
+The entrypoint `mocks.NewMockWorkspaceClient` exposes an API that allows you
+to retrieve a mock object, on which you can set expectations like so:
 
 ```go
-ctrl := gomock.NewController(t)
-defer ctrl.Finish()
+client := mocks.NewMockWorkspaceClient(t)
```

Comment:
please limit the changes only to readme.

Reply:
What should we do with the example? I think it refers to a service that has been renamed from dbfs to files maybe?